tools/wpt/expectation.json

```diff
@@ -3154,18 +3154,8 @@
         "queue-microtask.any.worker.html": true
       },
       "scripting": {
-        "reporterror.any.html": [
-          "self.reportError(1)",
-          "self.reportError(TypeError)",
-          "self.reportError(undefined)",
-          "self.reportError() doesn't invoke getters"
-        ],
-        "reporterror.any.worker.html": [
-          "self.reportError(1)",
-          "self.reportError(TypeError)",
-          "self.reportError(undefined)",
-          "self.reportError() doesn't invoke getters"
-        ]
+        "reporterror.any.html": false,
+        "reporterror.any.worker.html": false
```

Comment:
We can enable these by adding `e.preventDefault()` calls to https://github.com/web-platform-tests/wpt/blob/3518e60485b37ada91e3ab08daaa0591c4d16c4f/html/webappapis/scripting/reporterror.any.js, it wouldn't interfere with the tests. Not blocking though

Reply:
Let's do it. Do you want to open a PR to WPT, or shall I?

Reply:
> Let's do it. Do you want to open a PR to WPT, or shall I?

Uhh you can if you have it set up.

Reply:
https://github.com/web-platform-tests/wpt/pull/33657